File: app/models/appointment.rb
class Appointment < ApplicationRecord
  belongs_to :physician
  belongs_to :patient


  def self.all_and_then_some
    appointments = self.all
    return appointments.map do |appointment|
      {id: appointment.id, physician: appointment.physician, patient: appointment.patient}
    
    # binding.pry
    end
  end
end

File: client/src/components/Appointment.js
import Card from "./Card"

const Appointment = ({id,patient,physician, date}) =>{
    return (
        <Card>
            <h4>Patient: {patient.name} id:{patient.id}</h4>
            <h4>Physician: {physician.name} id:{physician.id}</h4>
            <p>Appointments: {date}</p>
            <p>Appointment id: {id}</p>
  
        </Card>
    )
}

export default Appointment